packages/docs/src/page-configs/getting-started/tree-shaking/code-examples/css-code-split.ts

```diff
@@ -0,0 +1,8 @@
+export const cssCodeSplit = `
+// Required
+import 'vuestic-ui/styles/essential.css'
+// Optional
+import 'vuestic-ui/styles/typography.css'
+import 'vuestic-ui/styles/grid.css'
```

Comment:
`.mx-0` is not exactly grid, rather spacers. But probably fine for now as we're gonna overhaul these regardless.

Reply:
Yeah. Let's make more CSS modules later.